CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1 @@
-- FIX Update IOTA to the last version of the library.
-- ADD bin section to the package.json to publish commands.
-- ADD Configuration selection in executable.
-- ADD Special configuration for tests.
\ No newline at end of file
+- Implement generic module in the parser (#21).
```

Comment:
We usually use the first verb to indicate if its an "Add", "Remove", "Fix" and so on, but quite liberally, actually. I'll change the "Implement" to "Add" for coherence with the rest of the software. I try to follow Orion's style in general, as it is the most stable project in our repos.